en/components/grid.md

```diff
@@ -639,3 +638,16 @@ Our community is active and always welcoming to new ideas.
 
 * [Ignite UI for Angular **Forums**](https://www.infragistics.com/community/forums/f/ignite-ui-for-angular)
 * [Ignite UI for Angular **GitHub**](https://github.com/IgniteUI/igniteui-angular)
+
+## Grid Limitations
+
+<div ckass="devider"></div>
```

Comment:
Class is misspelled and I'm not sure if the class name is spelled properly either. :)